atomics/t1121/t1121.yaml

```diff
@@ -0,0 +1,38 @@
+attack_technique: T1121
+display_name: RegSvcs/RegAsm
+
+atomic_tests:
+- name: Regasm Uninstall Method Call Test
+  description: |
+    Executes the Uninstall Method, No Admin Rights Required
+  supported_platforms:
+    - windows
+  input_arguments:
```

Comment:
I think we need a pre-requisites argument that tells someone how to build the CS file into the DLL required by the test.

Maybe something like
```
prerequisites:
  msbuild:
    command: msbuild.exe
```

What do you think?